<?xml version="1.0"?>
<launch>
    <node pkg="cob_mmcontroller" type="cob_config_controller" name="arm_mmcontroller_node" ns="arm_controller" cwd="node" respawn="false" output="log">
        <remap from="/arm_controller/cart/command" to="/cmd_vel"/>
        <param name="arm_base" value="arm_0_link" />
        <param name="arm_end_effector" value="sdh_tip_link" />
        <param name="default_control_mode" value="arm_base" />
        <param name="base_to_arm_ratio" value="0.5" />
    </node>

    <node pkg="cob_articulation_controller" type="cob_cartesian_trajectories_PID" name="cob_cartesian_trajectories_PID" cwd="node" respawn="false" output="screen">
        <param name="p_gain" value="1.0" />
        <param name="i_gain" value="0.000" /> <!--"0.0002" /-->
        <param name="d_gain" value="0.001" />
    </node>
        
    <node pkg="articulation_models" type="model_learner_prior" name="model_learner_prior" cwd="node"/>

    <node pkg="cob_articulation_controller" type="cob_articulation_cartcollector_prior.py" name="cob_articulation_cartcollector_prior" cwd="node"/>
        <!--param name="sigma_orientation" value="0.5" /-->

    <node pkg="cob_articulation_controller" type="cob_learn_model_prior.py" name="cob_learn_model_prior" cwd="node" output="screen"/>

</launch>
